tests/test_snapshot_is_pto_score.py

```diff
@@ -87,4 +89,3 @@ def test_snapshot_accuracy(snapshot):
 
     #Creating snapshot directory
     snapshot.assert_match(f"{accuracy},{precision},{recall},{f1_score}","overall_score.txt")
```

Comment:
@ajitava-git Did we run this code through pylint?
I see some changes to the file formatting

Reply:
@avinash010 Yes, ran it through pylint.